source/extensions/bootstrap/wasm/config.h

```diff
@@ -19,21 +19,7 @@ namespace Wasm {
 using Common::Wasm::PluginHandleSharedPtrThreadLocal;
 using Envoy::Extensions::Common::Wasm::PluginHandleSharedPtr;
 using Envoy::Extensions::Common::Wasm::PluginSharedPtr;
-
-class WasmService {
-public:
-  WasmService(PluginSharedPtr plugin, PluginHandleSharedPtr singleton)
-      : plugin_(plugin), singleton_(std::move(singleton)) {}
-  WasmService(PluginSharedPtr plugin,
-              ThreadLocal::TypedSlotPtr<PluginHandleSharedPtrThreadLocal>&& tls_slot)
-      : plugin_(plugin), tls_slot_(std::move(tls_slot)) {}
-
-private:
-  PluginSharedPtr plugin_;
-  PluginHandleSharedPtr singleton_;
-  ThreadLocal::TypedSlotPtr<PluginHandleSharedPtrThreadLocal> tls_slot_;
-};
-
+using WasmService = Common::Wasm::PluginConfig;
```

Comment:
Since the type `WasmService` is only used in a few places, WDYT about changing this to `using Common::Wasm::PluginConfig` and using `PluginConfig` directly as the type name?